wherehows-web/app/components/visualization/charts/score-gauge.ts

```diff
@@ -119,24 +119,44 @@ export default class VisualizationChartsScoreGauge extends Component {
    */
   chartData: Array<IHighChartsDataConfig>;
 
-  constructor() {
-    super(...arguments);
-
+  /**
+   * Performs update functions for the charts upon initial and subsequent renders
+   */
+  updateChart(): void {
     const chartOptions = getBaseGaugeConfig();
     const chartData = getBaseChartDataConfig('score');
-    const maxScore = typeof this.maxScore === 'number' ? this.maxScore : 100;
-    const score = this.score || NaN;
+    const score = this.score || 0;
```

Comment:
If you set a default in the constructor, then you don't need to do this anywhere else in the class methods

Reply:
That was an accident woops